config/default.yml

```
@@ -80,3 +80,6 @@ MissingEnableComment:
 
 ParserBlockingJavaScript:
   enabled: true
+
+AssetSizeJavaScript:
+  enabled: false
```

Should add the default for `threshold_in_bytes` here too:

```suggestion
  enabled: false
  threshold_in_bytes: 10000
```